### DISCUSSION TITLE
俄烏兩軍交戰，Google 地圖暫停顯示交通路況保護當地人民

### DISCUSSION THREAD

**Discussion by pekoranoinu** [1101-GS4719-C/Python-Community-GS4719-C#382] [科技新知]
俄烏交戰之中，Google將google地圖功能暫停，以防俄國能夠查詢烏克蘭的交通狀況，掌控整個戰況。就算是戰爭也和科技脫不了關係，或許我們也能利用科技來幫助烏克蘭守護他們的家園。
https://technews.tw/2022/03/01/google-maps-temporarily-disables-live-traffic-data-in-ukraine/

**Comment by chupochun** [comment]
太有趣了 結合時事的分享 希望戰爭能盡早平息

**Comment by lx3211** [comment]
超連結直接丟好像會錯誤
點進去是404
`[](url)`
用這個語法試試？

**Reply by JunTingLin** [reply]
[俄烏兩軍交戰，Google 地圖暫停顯示交通路況保護當地人民](https://technews.tw/2022/03/01/google-maps-temporarily-disables-live-traffic-data-in-ukraine/)

**Comment by Ben6uo** [comment]
天祐烏克蘭,希望這是最後的一次戰爭,或者之後都能有一支駭客團隊駭掉想發動戰爭的國家的網絡

**Comment by 11timyyy** [comment]
很聰明的方法，或許也可以編輯錯誤的地圖來擾亂俄羅斯🤣

**Comment by sean1325418** [comment]
現代戰爭情報最重要，當然不能給入侵者這麼便利的工具

**Comment by yuchi06274** [comment]
現代戰爭瞬息萬變，情報掌握跟武器精良一樣重要，不知道戰爭何時能平息

**Comment by chia-kai-chang** [comment]
@110303504 讀完這篇科技新聞，有沒有得到甚麼啟發? 可以稍微歸納一下跟大家分享。

**Comment by cordydeng** [comment]
不得不說google高層的應變能力非常厲害，藉由科技的力量且不會傷害人的方法來對世界和平做出貢獻

**Comment by Lai27330** [comment]
情報在戰爭的地位並不低於武器的殺傷力 像這樣能不透過武力來保護人民的方法非常的重要 而且google這樣的大型企業也能盡自己的心力來貢獻出社會責任也是一件好事

**Comment by Jay-722** [comment]
在高科技的世界發生戰爭真的很恐怖，趕快結束

**Comment by etin2468** [comment]
最近兩國好像又開始要展開和平會談了，希望可以結束可怕的戰爭

**Comment by xkllkx** [comment]
希望世界和平，現在科技都那麼進步了，沒什麼是不能好好協商的吧，受苦的都是人民。

**Comment by bloodtrail0511** [comment]
希望俄軍能夠收手，各個公司都在運用他們的影響力希望平息戰爭，天佑烏克蘭
